### ISSUE TITLE
docker-compose.yaml cannot work from outside the local environment.

### ISSUE THREAD

**Issue by jainal09** [Tencent/Tendis#67]
## Description
I am following the docker compose cluster tutorial from here - http://tendis.cn/#/Tendisplus/%E8%BF%90%E7%BB%B4/new_cluster_compose?id=docker-compose%e7%89%88

It works as expected. But, i want to access this cluster from outside (i deployed this docker-compose in a vm)
But, when I try to access this from outside the vm through the vm's global ip it does not work because I believe in the docker compose file there are hard coded ip address and also subnet is also 192.168.98.0/24 so when accessing it from out side it gets forwarded to 192.168.98.12 but for a global network this is not possible as 192.168.98.12 is the local ip of the virtual machine.
## Expected Behavior
It should be docker host binded so when accessing from outside it should be redirected to host ip of the local machine.

## Current Behavior
mentioned in the desciption

## Possible Solution
something like this  --network: host should be added in networks of compose file.

## Steps to Reproduce (for bugs)
just run the docker-compose file on a cloud vm and try to access it from outside where the redis host is the global ip of the vm
eg.
My global ip of the cloud vm is - `40.114.118.11`
But when i try to access the cluster:
![image](https://user-images.githubusercontent.com/34179361/106350422-4a6e7b80-62fb-11eb-8101-e5a0eff10b74.png)

However from inside the vm:
![image](https://user-images.githubusercontent.com/34179361/106350460-873a7280-62fb-11eb-9fc3-cfec2cf9622c.png)

## Context
Trying to deploy the cluster for PROD.

## Your Environment

* Operating System and version: ubuntu 18
* Machine Specifications:  2vcpus 8gb ram 500 gb ssd
* Tendis Version:
* Tendis Configuration: docker-compose.yaml
* IO/Network used:
* Link to your project:

**Comment by jainal09** [comment]
@orangeagain @TendisDev

**Comment by OMG-By** [comment]
`docker-compose.yaml` is docker compose default  config file.

1.  if you  want cloud vm access cluster. you can use `redis-cli -h ip -p 30000 -a tendisplustest -c` to access. because  docker port 51002 mapping localhost 30000. can use `docker ps -a` look this.
2. if you want modify subnet. you can modify `docker-compose.yaml`  or use `helm` to build cluster.
3. --network may have safe problem. and cluster node port all is 51002. cluster will build fail.

**Comment by jainal09** [comment]
Yes, `redis-cli -h 40.114.118.11 -p 30000 -a tendisplustest -c` will solve the connection problem. 
But when i will run the set command it will redirect to 192.168.98.22:51002 which for the global tendis cluster deployment is incorrect what should I do in such condition.
Please see the below screen shot accessing a redis cluster located in a Vm with ip - 40.114.118.11
![image](https://user-images.githubusercontent.com/34179361/107871500-3cf2ed00-6ec8-11eb-90f2-cf9f191653c1.png)

**Comment by jainal09** [comment]
any updates on this @tencent-adm @OMG-By @vinchen @Evil-crow

**Comment by OMG-By** [comment]
In my env. it can work success.
![image](https://user-images.githubusercontent.com/17672351/108579831-cb9db900-7363-11eb-88a4-60829221a6a9.png)
![image](https://user-images.githubusercontent.com/17672351/108579837-d48e8a80-7363-11eb-8697-56b4cb5d32c8.png)

you can use `cluster nodes` and `cluster info` see cluster state.
@jainal09 What is your build process?

**Comment by OMG-By** [comment]
And If you've ever failed to build. you should rm tendis tmp data dir `/data/tmp/`.
This might affect the new cluster env.
